### PR TITLE
Readme.fr

### PR DESCRIPTION
**Pull request by B0pol**
#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [x] Meta improvement to the project (user facing)

#### Description of the changes in your PR

- record videos
- create clones
- take over the world
Translate README
#### Fixes the following issue(s)

- 

#### Relies on the following changes

- 

#### APK testing 

debug.zip

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

~~I still need to fix navigation links, but that will be easier with GitHub~~ done

### CODE REVIEW

**Comment by mhmdanas** [comment]
I'm making a PR to fix #5051, so I think you should wait until it is merged so that you can translate what I added too.

File: README.fr.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+NewPipe n'utilise aucune bibliothèque Google ni l'API YouTube. Les sites web sont uniquement analysés pour récupérer les informations nécessaires, pour que l'appli puisse être utilisée sur des appareils sans les services Google installés. De plus, pour utiliser NewPipe – un logiciel libre et « gauche d'auteur » _(copyleft)_  – vous n'avez pas besoin de compte YouTube. 
+
+### Fonctionnalités
+
+* Rechercher des vidéos
+* Afficher des informations générales à propos des vidéos
+* Regarder des vidéos YouTube
+* Écouter des vidéos YouTube
+* Lecteur flottant (aussi appelé lecteur popup)
+* Chosir le lecteur permettant de regarder des vidéos
```
Comment (pihug12) [inline README.fr.md:51]:
```suggestion
* Choisir le lecteur permettant de regarder des vidéos
```

File: README.fr.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+* Télécharger des vidéos
+* Télécharger uniquement l'audio
+* Ovurir une vidéo dans Kodi
```
Comment (pihug12) [inline README.fr.md:54]:
```suggestion
* Ouvrir une vidéo dans Kodi
```

File: README.fr.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+NewPipe supporte plusieurs services. Notre [documentation](https://teamnewpipe.github.io/documentation) donne plus d'informations à propos de l'ajout d'un service dans l'extracteur et l'appli. Veuillez nous contacter si vous avez l'intention d'ajouter un nouveau service. Les services actuellement supportés sont :
+
+* YouTube
+* SoundCloud \[beta\]
+* media.ccc.de \[beta\]
+* PeerTube \[beta\]
+
+## Mises à jour
+
+Lorsque des modifications sont effectués au code de NewPipe (soit grâce à de nouvelles nouveautés, soit à cause de corrections de bugs), une mise à jour finira par être disponible. Le format de version est x.xx.x . Pour obtenir les nouvelles versions, vous pouvez :
```
Comment (pihug12) [inline README.fr.md:84]:
```suggestion
Lorsque des modifications sont effectuées au code de NewPipe (soit grâce à de nouvelles nouveautés, soit à cause de corrections de bugs), une mise à jour finira par être disponible. Le format de version est x.xx.x . Pour obtenir les nouvelles versions, vous pouvez :
```

File: README.fr.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+ 1. Compiler l'application et créer un fichier APK vous-même. C'est la méthode la plus rapide, mais est plus compliquée, par conséquent nous vous conseillons d'utiliser une des autres méthodes.
+ 2. Ajouter notre dépôt F-Droid et installer NewPipe depuis celui-ci. Les mises à jour sont disponibles dès qu'elles sont publiées par l'équipe NewPipe. Veuillez lire les instructions pour ajouter notre dépôt F-Droid : https://newpipe.schabi.org/FAQ/tutorials/install-add-fdroid-repo/
+ 3. Télécharger depuis un fichier APK depus les [publications GitHub (releases)](https://github.com/TeamNewPipe/NewPipe/releases) et installez l'appli. Les mises à jour seront disponibles dès qu'elles sont publiées par l'équipe NewPipe.
```
Comment (pihug12) [inline README.fr.md:87]:
```suggestion
 3. Télécharger un fichier APK depuis les [publications GitHub (releases)](https://github.com/TeamNewPipe/NewPipe/releases) et installez l'appli. Les mises à jour seront disponibles dès qu'elles sont publiées par l'équipe NewPipe.
```

File: README.fr.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+ 4. Metre à jour via F-Droid. C'est la méthode la plus lente, puisque F-Droid doit être compiler et créer un fichier APK, le signer, puis publier la mise à jour à ses utilisateurs.
```
Comment (pihug12) [inline README.fr.md:88]:
```suggestion
 4. Mettre à jour via F-Droid. C'est la méthode la plus lente, puisque F-Droid doit compiler l'application, signer l'APK, puis publier la mise à jour à ses utilisateurs.
```

File: README.fr.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+    <td><a href="https://www.bountysource.com/teams/newpipe"><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/2/22/Bountysource.png/320px-Bountysource.png" alt="Bountysource" width="190px"></a></td>
+    <td><a href="https://www.bountysource.com/teams/newpipe"><img src="assets/bountysource_qr_code.png" alt="Voir NewPipe sur bountysource.com" width="100px"></a></td>
+    <td><a href="https://www.bountysource.com/teams/newpipe/issues"><img src="https://img.shields.io/bountysource/team/newpipe/activity.svg?colorB=cd201f" height="30px" alt="Regardez combien de bounties vous pouvez gagner."></a></td>
+  </tr>
+</table>
+
+## Politique de confidentialité
+
+Le projet NewPipe cherche a fournir une expérience anonyme et privée pour les services multimédias en ligne.
+Par conséquent, notre appli ne collecte rien sans votre consentement. La poliique de confidentialité de NewPipe explique en détail quelles données sont envoyées et conservées lorsque vous signaler une erreur, ou commentez sur notre blog. Vous pouvez trouver ce document [ici](https://newpipe.net/legal/privacy/).
```
Comment (pihug12) [inline README.fr.md:132]:
```suggestion
Par conséquent, notre appli ne collecte rien sans votre consentement. La politique de confidentialité de NewPipe explique en détail quelles données sont envoyées et conservées lorsque vous signalez une erreur, ou commentez sur notre blog. Vous pouvez trouver ce document [ici](https://newpipe.net/legal/privacy/).
```

File: README.fr.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+
+## Licence
+[![GNU GPLv3 Image](https://www.gnu.org/graphics/gplv3-127x51.png)](https://www.gnu.org/licenses/gpl-3.0.en.html)  
+
+NewPipe est un logiciel libre ; vous pouvez le redistribuer ou le modifier suivant les termes de 
+la [GNU General Public License](https://www.gnu.org/licenses/gpl.html) 
+telle que publiée par la Free Software Foundation , soit la version 3 de la Licence, soit (à votre gré) toute version ultérieure.
```
Comment (pihug12) [inline README.fr.md:139]:
```suggestion
telle que publiée par la Free Software Foundation, soit la version 3 de la Licence, soit (à votre gré) toute version ultérieure.
```

File: README.fr.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+1. Sauvegardez vos données via les Paramètres > Contenu -> Exporter la base de donnée, afin de conserver les abonnements, listes de lecture et l'historique
+2. Désinstallez NewPipe
+3. Téléchargez NewPipe depuis la nouvelle source et installer le
+4. Importez vos données précédemment exportées via les Paramètres > Contenu > Importer la base de données
+
+## Contribution
+Que ce soit pour des idées, des traductions, des changement de design, une amélioration du code ou un changement majeur du code, de l'aide est toujours appréciée.
+Plus on en fait, mieux c'est !
+
+Si vous aimeriez contribuer, veuiller regarder nos [notes de contribution](.github/CONTRIBUTING.md).
```
Comment (pihug12) [inline README.fr.md:102]:
```suggestion
Si vous aimeriez contribuer, veuillez regarder nos [notes de contribution](.github/CONTRIBUTING.md).
```

File: README.fr.md [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+Nous recommendons la méthode 2 à la plupart de nos utilisateurs. Les fichiers APK installés via la méthode 2 ou 3 sont compatibles, mais pas avec la méthode 4. Nous utilisons notre signature pour signer les fichiers APK, mais F-Droid utilise leur propre signature. Android compare les signatures lors des mises à jour pour des raisons de sécurité, voilà pourquoi ces méthodes sont incompatibles.
+
+Cependant, si vous voulez changer de source pour certaines raisons (par ex. une fonctionnalité principale de NewPipe ne fonctionne plus et F-Droid n'a pas encore publié la mise à jour), nous vous recommendons cette procédure :
+1. Sauvegardez vos données via les Paramètres > Contenu -> Exporter la base de donnée, afin de conserver les abonnements, listes de lecture et l'historique
+2. Désinstallez NewPipe
+3. Téléchargez NewPipe depuis la nouvelle source et installer le
+4. Importez vos données précédemment exportées via les Paramètres > Contenu > Importer la base de données
+
+## Contribution
+Que ce soit pour des idées, des traductions, des changement de design, une amélioration du code ou un changement majeur du code, de l'aide est toujours appréciée.
```
Comment (pihug12) [inline README.fr.md:99]:
```suggestion
Que ce soit pour des idées, des traductions, des changements de design, une amélioration du code ou un changement majeur du code, de l'aide est toujours appréciée.
```

File: README.fr.md [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+ 4. Metre à jour via F-Droid. C'est la méthode la plus lente, puisque F-Droid doit être compiler et créer un fichier APK, le signer, puis publier la mise à jour à ses utilisateurs.
+
+Nous recommendons la méthode 2 à la plupart de nos utilisateurs. Les fichiers APK installés via la méthode 2 ou 3 sont compatibles, mais pas avec la méthode 4. Nous utilisons notre signature pour signer les fichiers APK, mais F-Droid utilise leur propre signature. Android compare les signatures lors des mises à jour pour des raisons de sécurité, voilà pourquoi ces méthodes sont incompatibles.
+
+Cependant, si vous voulez changer de source pour certaines raisons (par ex. une fonctionnalité principale de NewPipe ne fonctionne plus et F-Droid n'a pas encore publié la mise à jour), nous vous recommendons cette procédure :
+1. Sauvegardez vos données via les Paramètres > Contenu -> Exporter la base de donnée, afin de conserver les abonnements, listes de lecture et l'historique
+2. Désinstallez NewPipe
+3. Téléchargez NewPipe depuis la nouvelle source et installer le
```
Comment (pihug12) [inline README.fr.md:95]:
```suggestion
3. Téléchargez NewPipe depuis la nouvelle source et installer l'application
```

**Review by pihug12 — CHANGES_REQUESTED** [review]
There are some typos.

File: README.fr.md (hunk 4 above)
Comment (TiA4f8R) [inline README.fr.md:87]:
```suggestion
 3. Télécharger un fichier APK depuis les [publications GitHub (releases)](https://github.com/TeamNewPipe/NewPipe/releases) et installer l'appli. Les mises à jour seront disponibles dès qu'elles sont publiées par l'équipe NewPipe.
```

File: README.fr.md [hunk 11]
```diff
@@ -0,0 +1,139 @@
...
+<hr>
+<p align="center"><a href="#Captures-décran">Captures d'écran</a> &bull; <a href="#description">Description</a> &bull; <a href="#fonctionnalités">Fonctionnalités</a> &bull; <a href="#mises-à-jour">Mises à jour</a> &bull; <a href="#contribution">Contribution</a> &bull; <a href="#donner">Donner</a> &bull; <a href="#licence">Licence</a></p>
+<p align="center"><a href="https://newpipe.net">Site web</a> &bull; <a href="https://newpipe.net/blog/">Blog</a> &bull; <a href="https://newpipe.net/FAQ/">FAQ</a> &bull; <a href="https://newpipe.net/press/">Presse</a></p>
+<hr>
+
+*Read this in other languages: [English](README.md), [Français](README.fr.md), [한국어](README.ko.md).*
+
+<b>AVERTISSEMENT : C'EST UNE VERSION BETA, PAR CONSÉQUENT VOUS POURRIEZ RENCONTRER DES BUGS. SI C'EST LE CAS, VEUILLEZ OUVRIR UN TICKET VIA NOTRE DÉPÔT GITHUB.</b>
+
+<b>METTRE NEWPIPE OU N'IMPORTE QUEL FORK SUR LE GOOGLE PLAY STORE VIOLE LEUR CONDITIONS GÉNÉRALES D'UTILISATION.</b>
```
Comment (TiA4f8R) [inline README.fr.md:23]:
```suggestion
<b>METTRE NEWPIPE OU N'IMPORTE QUEL FORK SUR LE GOOGLE PLAY STORE VIOLE LEURS CONDITIONS GÉNÉRALES D'UTILISATION.</b>
```

File: README.fr.md [hunk 12]
```diff
@@ -0,0 +1,139 @@
...
+ 2. Ajouter notre dépôt F-Droid et installer NewPipe depuis celui-ci. Les mises à jour sont disponibles dès qu'elles sont publiées par l'équipe NewPipe. Veuillez lire les instructions pour ajouter notre dépôt F-Droid : https://newpipe.schabi.org/FAQ/tutorials/install-add-fdroid-repo/
+ 3. Télécharger depuis un fichier APK depus les [publications GitHub (releases)](https://github.com/TeamNewPipe/NewPipe/releases) et installez l'appli. Les mises à jour seront disponibles dès qu'elles sont publiées par l'équipe NewPipe.
+ 4. Metre à jour via F-Droid. C'est la méthode la plus lente, puisque F-Droid doit être compiler et créer un fichier APK, le signer, puis publier la mise à jour à ses utilisateurs.
+
+Nous recommendons la méthode 2 à la plupart de nos utilisateurs. Les fichiers APK installés via la méthode 2 ou 3 sont compatibles, mais pas avec la méthode 4. Nous utilisons notre signature pour signer les fichiers APK, mais F-Droid utilise leur propre signature. Android compare les signatures lors des mises à jour pour des raisons de sécurité, voilà pourquoi ces méthodes sont incompatibles.
```
Comment (TiA4f8R) [inline README.fr.md:90]:
```suggestion
Nous recommandons la méthode 2 à la plupart de nos utilisateurs. Les fichiers APK installés via la méthode 2 ou 3 sont compatibles, mais pas avec la méthode 4. Nous utilisons notre signature pour signer les fichiers APK, mais F-Droid utilise sa propre signature. La création d'un APK via la méthode 1 exclut entièrement une signature. Android compare les signatures lors des mises à jour pour des raisons de sécurité, c'est pourquoi ces méthodes sont incompatibles.
```

File: README.fr.md [hunk 13]
```diff
@@ -0,0 +1,139 @@
...
+<a href="https://www.bountysource.com/teams/newpipe" alt="Bounties Bountysource"><img src="https://img.shields.io/bountysource/team/newpipe/activity.svg?colorB=cd201f"></a>
+</p>
+<hr>
+<p align="center"><a href="#Captures-décran">Captures d'écran</a> &bull; <a href="#description">Description</a> &bull; <a href="#fonctionnalités">Fonctionnalités</a> &bull; <a href="#mises-à-jour">Mises à jour</a> &bull; <a href="#contribution">Contribution</a> &bull; <a href="#donner">Donner</a> &bull; <a href="#licence">Licence</a></p>
+<p align="center"><a href="https://newpipe.net">Site web</a> &bull; <a href="https://newpipe.net/blog/">Blog</a> &bull; <a href="https://newpipe.net/FAQ/">FAQ</a> &bull; <a href="https://newpipe.net/press/">Presse</a></p>
+<hr>
+
+*Read this in other languages: [English](README.md), [Français](README.fr.md), [한국어](README.ko.md).*
+
+<b>AVERTISSEMENT : C'EST UNE VERSION BETA, PAR CONSÉQUENT VOUS POURRIEZ RENCONTRER DES BUGS. SI C'EST LE CAS, VEUILLEZ OUVRIR UN TICKET VIA NOTRE DÉPÔT GITHUB.</b>
```
Comment (TiA4f8R) [inline README.fr.md:21]:
```suggestion
<b>AVERTISSEMENT : C'EST UNE VERSION BÊTA, PAR CONSÉQUENT VOUS POURRIEZ RENCONTRER DES BOGUES. SI C'EST LE CAS, VEUILLEZ OUVRIR UN TICKET VIA NOTRE DÉPÔT GITHUB.</b>
```

File: README.fr.md (hunk 3 above)
Comment (TiA4f8R) [inline README.fr.md:84]:
```suggestion
Lorsque des modifications sont effectuées au code de NewPipe (soit grâce à de nouvelles nouveautés, soit à cause de corrections de bogues), une mise à jour finira par être disponible. Le format de version est x.xx.x . Pour obtenir les nouvelles versions, vous pouvez :
```

File: README.fr.md [hunk 14]
```diff
@@ -0,0 +1,139 @@
...
+* Chosir le lecteur permettant de regarder des vidéos
+* Télécharger des vidéos
+* Télécharger uniquement l'audio
+* Ovurir une vidéo dans Kodi
+* Afficher les vidéos suivantes / liées
+* Chercher sur YouTube dans une langue spécifique
+* Regarder / bloquer du contenu avec limite d'âge
+* Afficher des informations générales à propos des chaines
+* Rechercher des chaines
+* Regarder les vidéos d'une chaine
```
Comment (TiA4f8R) [inline README.fr.md:60]:
```suggestion
* Afficher des informations générales à propos des chaînes
* Rechercher des chaînes
* Regarder les vidéos d'une chaîne
```

File: README.fr.md [hunk 15]
```diff
@@ -0,0 +1,139 @@
...
+* Support d'Orbot / Tor (pas encore directement)
+* Support des résolutions 1080p / 2K / 4K
+* Regarder l'historique
+* S'abonner à des chaines
```
Comment (TiA4f8R) [inline README.fr.md:64]:
```suggestion
* S'abonner à des chaînes
```

File: README.fr.md [hunk 16]
```diff
@@ -0,0 +1,139 @@
...
+ 3. Télécharger depuis un fichier APK depus les [publications GitHub (releases)](https://github.com/TeamNewPipe/NewPipe/releases) et installez l'appli. Les mises à jour seront disponibles dès qu'elles sont publiées par l'équipe NewPipe.
+ 4. Metre à jour via F-Droid. C'est la méthode la plus lente, puisque F-Droid doit être compiler et créer un fichier APK, le signer, puis publier la mise à jour à ses utilisateurs.
+
+Nous recommendons la méthode 2 à la plupart de nos utilisateurs. Les fichiers APK installés via la méthode 2 ou 3 sont compatibles, mais pas avec la méthode 4. Nous utilisons notre signature pour signer les fichiers APK, mais F-Droid utilise leur propre signature. Android compare les signatures lors des mises à jour pour des raisons de sécurité, voilà pourquoi ces méthodes sont incompatibles.
+
+Cependant, si vous voulez changer de source pour certaines raisons (par ex. une fonctionnalité principale de NewPipe ne fonctionne plus et F-Droid n'a pas encore publié la mise à jour), nous vous recommendons cette procédure :
```
Comment (TiA4f8R) [inline README.fr.md:92]:
```suggestion
Cependant, si vous voulez changer de source pour certaines raisons (par ex. une fonctionnalité principale de NewPipe ne fonctionne plus et F-Droid n'a pas encore publié la mise à jour), nous vous recommandons cette procédure :
```

File: README.fr.md [hunk 17]
```diff
@@ -0,0 +1,139 @@
...
+1. Sauvegardez vos données via les Paramètres > Contenu -> Exporter la base de donnée, afin de conserver les abonnements, listes de lecture et l'historique
```
Comment (TiA4f8R) [inline README.fr.md:93]:
```suggestion
1. Sauvegardez vos données via les Paramètres > Contenu -> Exporter la base de données, afin de conserver les abonnements, les listes de lecture et l'historique
```

File: README.fr.md (hunk 10 above)
Comment (TiA4f8R) [inline README.fr.md:95]:
```suggestion
3. Téléchargez NewPipe depuis la nouvelle source et installez l'application
```

File: README.fr.md [hunk 18]
```diff
@@ -0,0 +1,139 @@
...
+Plus on en fait, mieux c'est !
+
+Si vous aimeriez contribuer, veuiller regarder nos [notes de contribution](.github/CONTRIBUTING.md).
+
+<a href="https://hosted.weblate.org/engage/newpipe/">
+<img src="https://hosted.weblate.org/widgets/newpipe/-/287x66-grey.png" alt="État de la traduction" />
+</a>
+
+## Donner
+Si vous aimez NewPipe, nous serions heureux de recevoir un don. Vous pouvez soit donner avec des bitcoin ou via Bountysource ou Liberapay. Pour plus d'informations, veuillez visiter notre [site web](https://newpipe.net/donate).
```
Comment (TiA4f8R) [inline README.fr.md:109]:
```suggestion
Si vous aimez NewPipe, nous serions heureux de recevoir un don. Vous pouvez soit donner avec des bitcoins, soit via Bountysource ou soit via Liberapay. Pour plus d'informations, veuillez visiter notre [site web](https://newpipe.net/donate).
```

File: README.fr.md [hunk 19]
```diff
@@ -0,0 +1,139 @@
...
+  <tr>
+    <td><a href="https://www.bountysource.com/teams/newpipe"><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/2/22/Bountysource.png/320px-Bountysource.png" alt="Bountysource" width="190px"></a></td>
+    <td><a href="https://www.bountysource.com/teams/newpipe"><img src="assets/bountysource_qr_code.png" alt="Voir NewPipe sur bountysource.com" width="100px"></a></td>
+    <td><a href="https://www.bountysource.com/teams/newpipe/issues"><img src="https://img.shields.io/bountysource/team/newpipe/activity.svg?colorB=cd201f" height="30px" alt="Regardez combien de bounties vous pouvez gagner."></a></td>
+  </tr>
+</table>
+
+## Politique de confidentialité
+
+Le projet NewPipe cherche a fournir une expérience anonyme et privée pour les services multimédias en ligne.
```
Comment (TiA4f8R) [inline README.fr.md:131]:
```suggestion
Le projet NewPipe cherche à fournir une expérience anonyme et privée pour les services multimédias en ligne.
```

File: README.fr.md [hunk 20]
```diff
@@ -0,0 +1,139 @@
+<p align="center"><a href="https://newpipe.net"><img src="assets/new_pipe_icon_5.png" width="150"></a></p> 
+<h2 align="center"><b>NewPipe</b></h2>
+<h4 align="center">Lecteur multimédia libre et léger pour Android.</h4>
+<p align="center"><a href="https://f-droid.org/fr/packages/org.schabi.newpipe/"><img src="https://gitlab.com/fdroid/artwork/-/raw/master/badge/get-it-on-fr-fr.png" width="172px"></a></p> 
+
+<p align="center">
+<a href="https://github.com/TeamNewPipe/NewPipe/releases" alt="GitHub release"><img src="https://img.shields.io/github/release/TeamNewPipe/NewPipe.svg" ></a>
+<a href="https://www.gnu.org/licenses/gpl-3.0" alt="Licence : GPLv3"><img src="https://img.shields.io/badge/License-GPL%20v3-blue.svg"></a>
+<a href="https://travis-ci.org/TeamNewPipe/NewPipe" alt="État de compilation"><img src="https://travis-ci.org/TeamNewPipe/NewPipe.svg"></a>
```
Comment (XiangRongLin) [inline README.fr.md:9]: This need to be replaced with the github badge.
see #5278

**Review by XiangRongLin — CHANGES_REQUESTED** [review]

File: README.fr.md [hunk 21]
```diff
@@ -0,0 +1,139 @@
...
+<a href="https://hosted.weblate.org/engage/newpipe/" alt="État de la traduction"><img src="https://hosted.weblate.org/widgets/newpipe/-/svg-badge.svg"></a>
+<a href="http://webchat.freenode.net/?channels=%23newpipe" alt="Salon IRC : #newpipe"><img src="https://img.shields.io/badge/IRC%20chat-%23newpipe-brightgreen.svg"></a>
+<a href="https://www.bountysource.com/teams/newpipe" alt="Bounties Bountysource"><img src="https://img.shields.io/bountysource/team/newpipe/activity.svg?colorB=cd201f"></a>
+</p>
+<hr>
+<p align="center"><a href="#Captures-décran">Captures d'écran</a> &bull; <a href="#description">Description</a> &bull; <a href="#fonctionnalités">Fonctionnalités</a> &bull; <a href="#mises-à-jour">Mises à jour</a> &bull; <a href="#contribution">Contribution</a> &bull; <a href="#donner">Donner</a> &bull; <a href="#licence">Licence</a></p>
+<p align="center"><a href="https://newpipe.net">Site web</a> &bull; <a href="https://newpipe.net/blog/">Blog</a> &bull; <a href="https://newpipe.net/FAQ/">FAQ</a> &bull; <a href="https://newpipe.net/press/">Presse</a></p>
+<hr>
+
+*Read this in other languages: [English](README.md), [Français](README.fr.md), [한국어](README.ko.md).*
```
Comment (TobiGr) [inline README.fr.md:19]: the newly added translations need to linked

**Review by TobiGr — CHANGES_REQUESTED** [review]

File: README.fr.md (hunk 14 above)
Comment (B0pol) [reply README.fr.md:60]: On n’est pas obligé https://fr.wiktionary.org/wiki/chaine

**Pull request closed without merging**